### DISCUSSION TITLE
Sophia Script results for non-administrator user

### DISCUSSION THREAD

**Discussion by denilenko** [farag2/Sophia-Script-for-Windows#443] [General]
I am run Sophia Script under a user with administrator rights. All OK. But many features, mostly "cosmetic", I need apply for ordinary user account, because working under administrator is insecure. How can I do that? Is that possible? When I run script under ordinary user, but it is not work (even if run "Run as Administrator").

**Comment by farag2** [comment]
Hi. This is the major Windows restriction. You can only apply settings to the current user or to HKLM. Also you cannot work under admin, the 2000s has passed. :))) You're an ordinary user with the ability to run apps as admin. Feel the difference)). This is the different. So it's fully secure to work this way. Trust me)). Just use your brain. malware or ransomware do not appear from the nowhere. You have to run it as admin (or enable a UAC pop-up to control it) according to manifest, unless nothing will make your PC any harm.

**Comment by denilenko** [comment]
Hi! You don't understand me (my english not very well, i know). I login system under ordinary user (for security reason, but I know admin password, and if I need tune something or install some app I can do it with "Run as Administrator"). How can I apply settings for current non-administrator user if even when I run script as admin it still showing warning LoggedInUserNotAdmin?

**Reply by farag2** [reply]
Ah, Now I undertood. Also I'm not a native too)). Well, in the scope of the script, you cannot run it because it checks user rights. So without changing the code you cannot bypass it. So I may suggest to do so: give the admin rights to the current user, log off from the admin account and log into that one and apply the script. Unless you cannot configure anything: the prevail functions require the admin rights. As I told previously, this is the Windows way how to configure system.

**Reply by farag2** [reply]
Or open the code and apply functions as is pointwise that are presented to HKCU only. But this is not very convenient. Returning to my statement: work as normal user with admin rights. There is nothing to fear at all. We're not in 2000s now. :)

**Reply by denilenko** [reply]
I thinking about it, but I thought that there are more elegant ways to apply settings for a ordinary user. Ok, I will try it tomorrow.

**Reply by farag2** [reply]
No way to it if there're also functions that require admin rights. You cannot code the script such way))
